scripts/utils/migration_scripts/cognito_cleardown/README.md

```diff
@@ -0,0 +1,24 @@
+dp-identity-api Cognito Clear down
+================
+
+### Purpose
+
+These processes are to be used to make an existing cognito userpool back to a clean state.  
+If using an awsb environment you need to aws `sso login --profile <environment aws profile name>`  
+To view and amend execution permissions  
+`ls –l [file_name]`  
+`chmod +x filename`  
+
+Please run in the given order...
+
+### Step 1 remove all user team/group membership
```

Comment:
You could combine steps 1 to 3 in make target, therefore developer less likely to make a mistake